config/routes.rb

```diff
@@ -5,8 +5,11 @@
   # root "articles#index"
 
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show], param: :id
-  end 
+    resources :posts, only: [:index, :show, :new, :create], param: :id
+    resources :comments, only: [:new, :create]
+    resources :likes, only: [:create]
+  end
+end 
```

Comment:
- [ ] Please make sure that your routes.rb file is setup correctly, I think you forget to add do after the post resources I'm getting this error.
![image](https://user-images.githubusercontent.com/74744787/222924404-bf4e72ec-a2a6-457e-9819-5794dd28f4ea.png)